src/Microsoft.ML.FastTree/GamClassification.cs

```diff
@@ -31,7 +31,9 @@
 namespace Microsoft.ML.Trainers.FastTree
 {
     public sealed class BinaryClassificationGamTrainer :
-    GamTrainerBase<BinaryClassificationGamTrainer.Options, BinaryPredictionTransformer<CalibratedPredictorBase>, CalibratedPredictorBase>
+        GamTrainerBase<BinaryClassificationGamTrainer.Options,
+        BinaryPredictionTransformer<CalibratedPredictorBase<BinaryClassificationGamModelParameters,PlattCalibrator>>,
+        CalibratedPredictorBase<BinaryClassificationGamModelParameters,PlattCalibrator>>
```

Comment:
>,P [](start = 70, length = 2)

Under default settings, visual studio's auto-formatting is I believe control-k-d.

Reply:
Right. Thanks!

---
In reply to: [254540313](https://github.com/dotnet/machinelearning/pull/2435#discussion_r254540313) [](ancestors = 254540313)